src/renderer/components/+apps-releases/release.store.ts

```diff
@@ -58,7 +58,7 @@ export class ReleaseStore extends ItemStore<HelmRelease> {
     this.isLoading = true;
     let items;
     try {
-      const { isAdmin, allowedNamespaces } = getHostedCluster();
+      const { isAdmin, allowedNamespaces } = getHostedCluster()
```

Comment:
why all the removing of semicolons?

Reply:
we don't have strict rules on that :) in general semicolon redundant and should be avoid imo..